### PR TITLE
Add strings for the tracker removal setting

### PR DESCRIPTION
**Pull request by Vinnl**
These are strings for the UI to enable/disable tracker removal in the settings page, i.e. this change: https://github.com/mozilla/fx-private-relay/pull/2067

Please do not merge this yet; there are more tracker removal-related strings to land.

image

image

### CODE REVIEW

**Review by peiying2 — APPROVED** [review]

**Comment by Vinnl** [comment]
@peiying2 I just added the rest of the strings!

Some screenshots:

image

image

image

image

image

File: en/app.ftl [hunk 1]
```diff
@@ -765,6 +799,43 @@ multi-part-onboarding-premium-extension-button-skip = Continue, I’ll download
...
+trackerreport-trackers-explainer-heading = About email trackers
+trackerreport-trackers-explainer-content-part1 = Email tracking is a common surveillance and advertising tool that has taken over many inboxes. These trackers can be used to understand more about your online behavior, your interests, and your email activity.
+trackerreport-trackers-explainer-content-part2 = A company or organization will embed a tracker in emails sent to you, usually hidden within an image or a link. When the email is opened, code within the tracker sends data back to the company.
+trackerreport-breakage-warning = Important: Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images, which will not load if they contain a tracker.
```
Comment (peiying2) [inline en/app.ftl:833]: Remove the comma before `because…`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Minor issues:
- stylistic on category names, capital initial for all? `Tracker removal` appears to be different from the others
- punctuation (how conditional clause is used)
- `-` n dash or hyphen.

File: en/app.ftl [hunk 2]
```diff
@@ -510,16 +529,26 @@ profile-label-copied = Copied!
 profile-label-blocked = Blocked
 profile-label-forwarded = Forwarded
 profile-label-replies = Replies
+# This is displayed in small under a number in a large font indicating the number of trackers that have been removed from all emails sent to a particular mask
+profile-label-trackers-removed = Trackers removed
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply en/app.ftl]: I think so, good eye! 6c0a6cb

File: en/app.ftl [hunk 3]
```diff
@@ -454,6 +465,14 @@ faq-question-promotional-email-blocking-answer = { -brand-name-relay-premium } s
 faq-question-detect-promotional-question = How does { -brand-name-relay } detect if an email is Promotional or not?
 faq-question-detect-promotional-answer = Many emails are sent with “header” metadata to indicate that they are from list-based automated tools. { -brand-name-firefox-relay } detects this header data so it can block these emails.
 
+faq-question-disable-trackerremoval-question = Can I stop removing email trackers?
+faq-question-disable-trackerremoval-answer = Yes. If you’re having trouble with emails looking broken or would like to stop removing trackers, you can disable the feature in settings.
+faq-question-bulk-trackerremoval-question = Can I remove trackers only on some of my email masks?
+faq-question-bulk-trackerremoval-answer = You can only turn tracker removal on at an account level—it either removes trackers from all of your emails, or none of them.
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply en/app.ftl]: b011408

File: en/app.ftl [hunk 4]
```diff
@@ -535,6 +564,9 @@ profile-filter-category-option-promo-blocking-masks = Promo-blocking masks
 profile-filter-category-option-random-masks = Random masks
 # Filter on Relay masks that have a user-chosen name (<user-chosen name>@<user-chosen subdomain>.mozmail.com).
 profile-filter-category-option-custom-masks = Custom masks
+# This is a button that, when clicked, will open a tooltip with profile-indicator-tracker-removal-tooltip ("Currently removing email trackers").
+profile-indicator-tracker-removal-alt = Tracker removal
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en/app.ftl:568]: Ah, I see. So where does this string appear? 
image

File: en/app.ftl (hunk 1 above)
Comment (Vinnl) [reply en/app.ftl]: 9913d3d

File: en/app.ftl (hunk 4 above)
[the remark this answers is not in the document]
Comment (Vinnl) [reply en/app.ftl]: Chris came up with an alternative: 95a57f0

The "Trackers removed" in your screenshot is `profile-label-trackers-removed`.

File: en/app.ftl [hunk 5]
```diff
@@ -72,7 +72,7 @@ nav-settings = Settings
 nav-support = Help and Support
 nav-sign-out = Sign Out
 nav-contact = Contact Us
- 
+
```
Comment (peiying2) [inline en/app.ftl:75]: this change seems to introduce an error? See the commit with an `x` status (the last of a series of commits). Is it the line break breaks the section under `## Mobile menu text`? Not sure.

**Comment by peiying2** [comment]
@Vinnl Take a look of this file and identify the issues that failed linter:

1) https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/3e79fa545a05c86d6e1327dbde5e66de588ad19b/en/app.ftl#L494. You have an extra line break before the ## comment. Should be just one line space.

2) https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/3e79fa545a05c86d6e1327dbde5e66de588ad19b/en/app.ftl#L826 - notice the string ID contains `_` rather than `-`.

**Review by peiying2 — CHANGES_REQUESTED** [review]
The linter script will show the types of problems that failed the check: https://github.com/mozilla-l10n/moz-fluent-linter/blob/main/src/fluent_linter/linter.py. I commented on where the issues are. Please make changes accordingly. Thank you!

**Comment by Vinnl** [comment]
Thanks for explaining the linter errors @peiying2, fixed in 76a5f5f.

**Review by peiying2 — APPROVED** [review]
LGTM!!

**Comment by Vinnl** [comment]
Thanks for all the improvements @peiying2!

**Pull request merged**